src/ui/tabs/history_tab.py

```
@@ -733,209 +788,195 @@ def get_csv_value(row_dict, key_name, default_if_missing_or_empty):
+            try:
+                grp = grouped.setdefault(key, {"house":0.0,"water":0.0,"gas":0.0,"unit":0.0})
+                grp["house"] += float(room_data.get("house_rent", 0) or 0)
+                grp["water"] += float(room_data.get("water_bill", 0) or 0)
+                grp["gas"]   += float(room_data.get("gas_bill", 0) or 0)
+                grp["unit"]  += float(room_data.get("unit_bill", 0) or 0)
+            except (ValueError, TypeError):
+                continue
+
+        # sort keys by year then month
+        def month_key(m):
+            parts=m.split();
```

Ruff (0.11.9)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)